starwars/src/components/StarWarsList.js

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useState } from "react"
 import axios from "axios"
 import StarWarsCard from "./StarWarsCard"
-import { Container, Row } from "reactstrap"
+import { Container } from "reactstrap"
 
 export default function StarWarsList() {
     const [chars, setChars] = useState([])
```

Comment:
Good use of state here. Initializing state as an array and setting it as an array of objects is the best way to handle this.